src/index.css

```diff
@@ -0,0 +1,129 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: orangered;
+}
+
+.header {
+  height: 30vh;
+  width: 100%;
```

Comment:
Good job on your project, it looks awesome. However there is one thing you have overlooked, which is the correct font family  (Lato) for the website. Please use the fonts as specified in the project descriptions in order to make your design match with the template

Reply:
Thanks
I will update it.